File: TestAutomation/pages/PostPage.cs
using OpenQA.Selenium;

namespace TestAutomation.pages
{
    internal class PostPage : Page
    {
        public PostPage(IWebDriver driver) : base(driver)
        {
        }

        internal string getPageTitle()
        {
            return driver.FindElement(By.ClassName("entry-title")).Text;
        }

        internal string getPageContent()
        {
            return driver.FindElement(By.XPath("//div[@class='entry-content']/p[1]")).Text;
        }
    }
}

File: src/main/java/Pages/PostPage.java
package Pages;

import Pages.Page;
import Pages.PostPreviewPage;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

/**
 * Created by Administrator on 2016-09-23.
 */
public class PostPage extends Page {
    public PostPage(WebDriver driver) {
        super(driver);

    }

    public void addNewPost(String title) throws InterruptedException {


         Thread.sleep(3000);
        xpathSendKeyTitle(".//*[@id='primary']/div/div[2]/div[1]/div[2]/div[3]/div/textarea", title); // potzrena dwoch wartosci do moetody

    }



    public PostPreviewPage goToPostPreviewPage() throws InterruptedException {
        Thread.sleep(1000);
        click(By.xpath(" .//*[@id='primary']/div/div[2]/div[2]/div[2]/div[1]/div[3]/div/button[1]"));

        return new PostPreviewPage(driver);
    }


}
